test/Faker/Provider/ne_NP/PaymentTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Faker\Test\Provider\ne_NP;
+
+use Faker\Provider\ne_NP\Payment;
+use Faker\Test\TestCase;
+
+final class PaymentTest extends TestCase
+{
+    public function testCommercialBank(): void
+    {
+        $str = $this->faker->commercialBank();
+        self::assertIsString($str);
```

Comment:
Please add the specific bank to the test that should be returned. We provide a seed to the provider.